### PR TITLE
Small Tweaks

### PR DESCRIPTION
**Pull request by dvaldivia**
These are a bunch of tweaks we did for last week

### CODE REVIEW

File: portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx [hunk 1]
```diff
@@ -124,8 +144,26 @@ const ListTenants = ({ classes }: ITenantsList) => {
...
+  };
+
+  const openLinkCloud = (element: ITenant) => {
+    const link = `https://${element.name}.cloud.min.dev`;
+
+    openLink(link);
+  };
+
+  const openLinkConsole = (element: ITenant) => {
+    const link = `https://console.${element.name}.cloud.min.dev`;
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx:158]: Need to remove these hardcoded links

File: portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx [hunk 2]
```diff
@@ -124,8 +144,26 @@ const ListTenants = ({ classes }: ITenantsList) => {
     setRowsPerPage(rPP);
   };
 
+  const openLink = (link: string) => {
+    window.open(link, "_blank");
+  };
+
+  const openLinkCloud = (element: ITenant) => {
+    const link = `https://${element.name}.cloud.min.dev`;
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx:152]: Need to remove these hardcoded links

File: portal-ui/src/index.tsx [hunk 3]
```diff
@@ -33,34 +34,35 @@ const GlobalCss = withStyles({
...
+      fontSize: "0.8125rem",
     },
     ".MuiTableCell-head": {
       borderRadius: "3px 3px 0px 0px",
-      fontSize: "13px"
+      fontSize: "13px",
     },
     ".MuiPaper-root": {
-      borderRadius: "3px"
+      borderRadius: "3px",
```
Comment (bexsoft) [inline portal-ui/src/index.tsx:48]: This needs to be an integer

File: portal-ui/src/index.tsx [hunk 4]
```diff
@@ -33,34 +34,35 @@ const GlobalCss = withStyles({
...
     },
     ".MuiButton-sizeSmall": {
       padding: "4px 10px",
-      fontSize: "0.8125rem"
+      fontSize: "0.8125rem",
     },
     ".MuiTableCell-head": {
       borderRadius: "3px 3px 0px 0px",
-      fontSize: "13px"
+      fontSize: "13px",
```
Comment (bexsoft) [inline portal-ui/src/index.tsx:45]: This needs to be an integer

File: portal-ui/src/index.tsx [hunk 5]
```diff
@@ -33,34 +34,35 @@ const GlobalCss = withStyles({
       fontSize: "14px",
       textTransform: "capitalize",
       padding: "16px 25px 16px 25px",
-      borderRadius: "3px"
+      borderRadius: "3px",
```
Comment (bexsoft) [inline portal-ui/src/index.tsx:37]: This needs to be an integer

File: portal-ui/src/index.tsx [hunk 6]
```diff
@@ -33,34 +34,35 @@ const GlobalCss = withStyles({
...
-      borderRight: "0px"
+      borderRight: "0px",
     },
     ".MuiDrawer-root": {
       "& .MuiPaper-root": {
-        borderRadius: "0px"
-      }
-    }
-  }
+        borderRadius: "0px",
```
Comment (bexsoft) [inline portal-ui/src/index.tsx:55]: This needs to be an integer

File: portal-ui/src/screens/Console/Common/FormComponents/common/styleLibrary.ts [hunk 7]
```diff
@@ -20,7 +20,7 @@ export const fieldBasic = {
   inputLabel: {
     fontWeight: 500,
     marginRight: 10,
-    width: 100,
+    width: "160px",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Common/FormComponents/common/styleLibrary.ts:23]: This needs to be an integer

File: portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx (hunk 2 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx:152]: We may want to re-add it later on?

**Pull request merged**